src/writer/xapianIndexer.cpp

```diff
@@ -89,10 +94,12 @@ void XapianIndexer::indexingPrelude()
     case IndexingMode::TITLE:
       writableDatabase.set_metadata("valuesmap", "title:0;targetPath:1");
       writableDatabase.set_metadata("kind", "title");
+      writableDatabase.set_metadata("data", "fullPath");
       break;
     case IndexingMode::FULL:
       writableDatabase.set_metadata("valuesmap", "title:0;wordcount:1;geo.position:2");
       writableDatabase.set_metadata("kind", "fulltext");
+      writableDatabase.set_metadata("data", "fullPath");
```

Comment:
This commit (and all other commit "reading" the metadata and the dbData) can be merge with the commit adding the "C/" prefix.